src/smtsolvers/CoreSMTSolver.h

```diff
@@ -651,35 +648,22 @@ inline bool     CoreSMTSolver::enqueue         (Lit p, CRef from)
     return value(p) != l_Undef ? value(p) != l_False : (uncheckedEnqueue(p, from), true);
 }
 
-inline bool     CoreSMTSolver::addOriginalClause(const vec<Lit> & ps)
+inline bool     CoreSMTSolver::addOriginalClause(vec<Lit> && ps)
 {
-    return addOriginalClause_(ps);
+    return addOriginalClause_(std::move(ps));
 }
-inline bool     CoreSMTSolver::addEmptyClause  ()
```

Comment:
addEmptyClause is no longer needed?
Mb also remove it in Solver.h and SimpSolver.h?

Reply:
It was not used anywhere.
`Solver.h` and `SimpSolver.h` are MiniSat files that are not currently in use.
No reason to go and make changes in those.

Reply:
Ok, makes sense, will we need to use them though?
Or they should be refactored/removed totally(as a general knowledge)

Reply:
We can discuss if we should remove MiniSAT files that we do not use. They are there for historical reason, but maybe we can remove them now.